packages/common/client-utils/package.json

```diff
@@ -0,0 +1,154 @@
+	"browser": {
+		"./dist/indexNode.js": "./dist/indexBrowser.js",
+		"./lib/indexNode.js": "./lib/indexBrowser.js"
+	},
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "fluid-build . --task build",
+		"build:commonjs": "fluid-build . --task commonjs",
+		"build:compile": "fluid-build . --task compile",
+		"build:docs": "api-extractor run --local --typescript-compiler-folder ./node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
+		"build:esnext": "tsc --project ./tsconfig.esnext.json",
+		"build:test": "concurrently npm:build:test:mocha npm:build:test:jest npm:build:test:types",
+		"build:test:jest": "tsc --project ./src/test/jest/tsconfig.json",
+		"build:test:mocha": "tsc --project ./src/test/mocha/tsconfig.json",
+		"build:test:types": "tsc --project ./src/test/types/tsconfig.json",
+		"ci:build": "npm run build:compile",
+		"ci:build:docs": "api-extractor run --typescript-compiler-folder ./node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
+		"ci:test": "npm run test:report",
+		"ci:test:coverage": "npm run test:coverage",
+		"clean": "rimraf --glob _api-extractor-temp dist lib *.tsbuildinfo *.build.log",
+		"eslint": "eslint --format stylish src",
+		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+		"format": "npm run prettier:fix",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check . --ignore-path ../../../.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
+		"test": "npm run test:mocha && npm run test:jest",
+		"test:coverage": "npm run test:report",
+		"test:jest": "jest",
+		"test:jest:report": "npm run test:jest -- --ci --coverage --reporters=default --reporters=jest-junit",
+		"test:mocha": "mocha --unhandled-rejections=strict  --recursive dist/test/mocha/**/*.spec.js --exit --project test/tsconfig.json",
+		"test:mocha:multireport": "cross-env FLUID_TEST_MULTIREPORT=1 npm run test:mocha",
+		"test:mocha:report": "npm run test:mocha -- -- --reporter xunit --reporter-option output=nyc/mocha-junit-report.xml",
+		"test:report": "nyc npm run test:mocha:report && npm run test:jest:report",
+		"tsc": "tsc",
+		"typetests:gen": "fluid-type-test-generator",
+		"typetests:prepare": "flub typetests --dir . --reset --previous --normalize"
+	},
+	"nyc": {
+		"all": true,
+		"cache-dir": "nyc/.cache",
+		"exclude": [
+			"src/test/**/*.ts",
+			"dist/test/**/*.js"
+		],
+		"exclude-after-remap": false,
+		"include": [
+			"src/**/*.ts",
+			"dist/**/*.js"
+		],
+		"report-dir": "nyc/report",
+		"reporter": [
+			"cobertura",
+			"html",
+			"text"
+		],
+		"temp-directory": "nyc/.nyc_output"
+	},
+	"dependencies": {
+		"@fluidframework/core-interfaces": "workspace:~",
+		"@fluidframework/core-utils": "workspace:~",
+		"@types/events": "^3.0.0",
```

Comment:
This should be a dev dep, right?

Reply:
You mean the events types? No, we need those in deps because the types are part of our exported APIs.